agenda-services/src/main/java/org/exoplatform/agenda/notification/builder/AgendaTemplateBuilder.java

```diff
@@ -124,7 +126,7 @@ protected MessageInfo makeMessage(NotificationContext ctx) {
                                                                 notification,
                                                                 timeZone);
       MessageInfo messageInfo = buildMessageSubjectAndBody(templateContext, notification, pushNotificationURL);
-      addIcsFile(notification, messageInfo);
+      addIcsFile(notification, messageInfo,timeZone);
```

Comment:
```suggestion
      addIcsFile(notification, messageInfo, timeZone);
```
please, try to format the added code (not the whole file)